cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/ApplyLogic.java

```diff
@@ -51,7 +51,8 @@ public void applyApplications(@Nonnull Map<String, ApplicationBean> desiredAppli
 
         ApplicationsOperations applicationsOperations = new ApplicationsOperations(cfOperations);
         log.info("Fetching information about applications...");
-        Map<String, ApplicationBean> liveApplications = applicationsOperations.getAll().block();
+        GetLogic getLogic = new GetLogic();
+        Map<String, ApplicationBean> liveApplications = getLogic.getApplications(applicationsOperations);
```

Comment:
Looks like this had been forgotten in the last sprint?

Reply:
Yes it has. It was inconsistent to other places where the GetLogic has already been used

Reply:
Makes sense, as we discussed earlier.